src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (UserService.findByEmail(email) != null
```

Comment:
I would strongly recommend to use `UserService` as class member of `AuthenticationService`. Also, I would say you should firstly fetch user like
`User user = userService.findByEmail(email);`
And after that do all checks.